tests/unit_tests/test_tethys_apps/test_app_installation.py

```diff
@@ -12,8 +12,11 @@
 
 class TestAppInstallation(unittest.TestCase):
     def setUp(self):
-        self.src_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
-        self.root = os.path.join(self.src_dir, 'tethys_apps', 'tethysapp', 'test_app', 'public')
+        self.src_dir = os.path.dirname(os.path.dirname(
```

Comment:
I'm seeing a lot of formatting changes related to column width. We use a column width of 120 for Tethys development. I don't think many of these changes are necessary. Can you revert?